apps/perf-test-react-components/package.json

```diff
@@ -22,7 +22,6 @@
     "@fluentui/react-avatar": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29603&commitId=818a99ba99de96825acd4c3131b6e47eaae5ca05&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.size- large.chromium.png|0| Added|
|InfoLabel.size- large.rest.chromium.png|0| Added|
|InfoLabel.size- medium.chromium.png|0| Added|
|InfoLabel.size- medium.rest.chromium.png|0| Added|
|InfoLabel.size- small.chromium.png|0| Added|
|InfoLabel.size- small.rest.chromium.png|0| Added|
|InfoLabel.size-large.rest.chromium.png|0| Removed|
|InfoLabel.size-small.rest.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Expected changes